web-console-sveltekit/README.md

```diff
@@ -81,4 +81,4 @@ bun run generate-openapi
 - `src/lib/services/`: Functions that describe side effects (persistent storage, networking etc.)
 - `src/lib/types/`: Types used throughout the app
 - `src/routes/`: Web app pages used in file-based routing
-- `static/`: Static assets served as-is
\ No newline at end of file
+- `static/`: Static assets served as-is
```

Comment:
Adding general comment here but we should rm -rf web-console and mv web-console-sveltekit to web-console